docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql.md

```diff
@@ -14,7 +14,7 @@ The SQL database introspection and partitioning are useful for:
 - Exploring the schema and column metadata of the tables in your SQL database, and
 - Organizing the tables into <TechnicalTag tag="data_asset" text="Data Assets" /> according to the partitioning considerations informed by this exploration.
 
-Partitioning enables you to select the desired subsets of your dataset for [Validation](../../reference/validation.md).
+Partitioning enables you to select the desired subsets of your dataset for [Validation](../../terms/checkpoint.md).
```

Comment:
As with the previous comment -- if this is linking to the checkpoint page, the link should mention checkpoints.